fixuid.go

```diff
@@ -183,10 +183,56 @@ func main() {
 		}
 	}
 
+	// call syscall.Setgroups if necessary
+	if oldUID != newUID || oldGID != newGID {
```

Comment:
IMHO should be run in any case.

Reply:
The container start process sets groups. If no UIDs have changed, all of the groups should already be correct